### PR TITLE
OHRM5X-1282:  Url link validation

### PR DESCRIPTION
**Pull request by Ajiharan007**

### CODE REVIEW

File: installer/Util/AppSetupUtility.php [hunk 1]
```diff
@@ -59,6 +59,9 @@ class AppSetupUtility
             \OrangeHRM\Installer\Migration\V5_0_0_beta\Migration::class,
             \OrangeHRM\Installer\Migration\V5_0_0\Migration::class,
         ],
+        '5.1' => [
+            \OrangeHRM\Installer\Migration\V5_1_0\Migration::class,
+        ],
```
Comment (RajithaKumara) [inline installer/Util/AppSetupUtility.php:64]:
```suggestion
        '5.1' => \OrangeHRM\Installer\Migration\V5_1_0\Migration::class,
```

File: installer/Migration/V5_1_0/Migration.php [hunk 2]
```diff
@@ -0,0 +1,49 @@
...
+    {
+        return '5.1.0';
+    }
+
+    /**
+     * @return void
+     */
+    private function addValidColumnToRequestResetPassword(): void
+    {
+        $this->getSchemaHelper()->addColumn(
```
Comment (RajithaKumara) [inline installer/Migration/V5_1_0/Migration.php:42]: @SharukaNP how to define boolean column. can you check DBAL docs

**Review by RajithaKumara — CHANGES_REQUESTED** [review]

File: installer/Migration/V5_1_0/Migration.php (hunk 2 above)
Comment (Ajiharan007) [reply installer/Migration/V5_1_0/Migration.php:53]: changed

File: installer/Migration/V5_1_0/Migration.php [hunk 3]
```diff
@@ -31,6 +32,7 @@ class Migration extends AbstractMigration
     public function up(): void
     {
         $this->getDataGroupHelper()->insertApiPermissions(__DIR__ . '/permission/api.yaml');
+        $this->addValidColumnToRequestResetPassword();
 
         $this->getConnection()->executeStatement(
             'ALTER TABLE ohrm_kpi CHANGE job_title_code job_title_code INT(13) NOT NULL'
```
Comment (RajithaKumara) [inline installer/Migration/V5_1_0/Migration.php:38]: @SharukaNP don't we have DBAL solution

**Review by RajithaKumara — APPROVED** [review]

File: installer/Migration/V5_1_0/Migration.php [hunk 4]
```diff
@@ -81,6 +83,22 @@ public function up(): void
         }
     }
 
+    /**
+     * @return void
+     */
```
Comment (devishke-orange) [inline installer/Migration/V5_1_0/Migration.php:88]: remove return void

File: src/plugins/orangehrmAuthenticationPlugin/Dao/ResetPasswordDao.php [hunk 5]
```diff
@@ -34,6 +34,7 @@ public function saveResetPasswordRequest(ResetPasswordRequest $resetPassword): R
         return $resetPassword;
     }
 
+
     /**
```
Comment (devishke-orange) [inline src/plugins/orangehrmAuthenticationPlugin/Dao/ResetPasswordDao.php:38]: remove extra line
```suggestion
    }

    /**
```

**Review by devishke-orange — CHANGES_REQUESTED** [review]
Looks good :+1: just two formatting suggestions

**Pull request merged**